public/js/retraining.jsx

```diff
@@ -254,7 +254,7 @@ class WindowShade extends React.Component {
 class UpdateForm extends React.Component {
   constructor() {
     super();
-    this.state = { classCount: 0, files: {}, showFlash: false, checkboxes: new Set([]) };
```

Comment:
Doh! I thought I undid all of those. That was webstorm getting a little too aggressive with a rename. :(